README.txt

```diff
@@ -43,9 +43,9 @@ Default controls
 - 0-9: Select item
 - Z: Zoom (needs zoom privilege)
 - T: Chat
-- /: Commad
+- /: Command
 
-- Esc: Pause menu/abort/exit (pauses only singleplayer game)
+- Esc: Pause menu/abort/exit (pauses only single-player game)
```

Comment:
`singleplayer` is used everywhere: [Dev Wiki search results](http://dev.minetest.net/index.php?search=singleplayer&go=Go&title=Special%3ASearch)